README.md

```diff
@@ -22,6 +22,19 @@ cd jr
 make release.linux -j
 ```
 
+This is producing an executable named "jerry" in "build/bin/release.linux/".
+
+### Hello world
```

Comment:
Well, actually, it is not a hello _world_ :). "Running your First Script"? Or something like that.